files/image_config/rsyslog/rsyslog.conf.j2

```diff
@@ -59,6 +64,32 @@ $UDPServerRun 514
 $template SONiCFileFormat,"%timegenerated%.%timegenerated:::date-subseconds% %HOSTNAME% %syslogseverity-text:::uppercase% %syslogtag%%msg:::sp-if-no-1st-sp%%msg:::drop-last-lf%\n"
 $ActionFileDefaultTemplate SONiCFileFormat
 
+template(name="WelfRemoteFormat" type="string" string="%TIMESTAMP% id=firewall time=\"%timereported\
+:::date-year%-%timereported:::date-month%-%timereported:::date-day% %timereported:::date-hour%:%timereported:::date-minute%:%timereported\
+:::date-second%\" fw=\"{{ fw_name }}\" pri=%syslogpriority% msg=\"%syslogtag%%msg:::sp-if-no-1st-sp%%msg:::drop-last-lf%\"\n")
+
+#Set remote syslog server
+{% for server in SYSLOG_SERVER %}
+{% set conf = SYSLOG_SERVER[server] | d({}) -%}
+
```

Comment:
Why do we need defaults file. Can't we use init_cfg.json to add the syslog defaults?

Reply:
We use the defaults file for other features as well
It is very useful because with this single file is used in multiple j2 files
For rsyslog it is not necessary but it's a good improvement  for future

Reply:
default file removed